compiler/x/codegen/OMRX86Instruction.cpp

```diff
@@ -3439,62 +3439,62 @@ TR::InstOpCode::Mnemonic getBranchOrSetOpCodeForFPComparison(TR::ILOpCodes cmpOp
 
       case TR::iffcmpleu:
       case TR::ifdcmpleu:
-         op = useFCOMIInstructions ? TR::InstOpCode::JBE4 : TR::InstOpCode::JNE4;
+         op = TR::InstOpCode::JBE4;
```

Comment:
Maybe I'm misunderstanding, but FCOMI instructions are x87 instructions right? If the compiler can't use the FCOMI instructions, then shouldn't we be using the falsy expression (ie `TR::InstOpCode::JNE4`)? Same for all the changes below.

Reply:
Ah I see, I got confused and thought the FCOMI instructions were x87 because queries around them were also removed, but I guess these were removed because there's no point querying the default behaviour.